main.py

```diff
@@ -181,6 +181,10 @@ def notify_uplink_detected(self, event):
             interface_a = switch_a.get_interface_by_port_no(port_a.value)
             interface_b = switch_b.get_interface_by_port_no(port_b.value)
 
+            # Return if any interface does not exist
+            if not (interface_a and interface_b):
+                return
```

Comment:
Are you 100% sure it should not generate any event here?

Reply:
I think this check is no longer needed and this PR can be closed now, because this problem is solved by the PR https://github.com/kytos/topology/pull/127.